src/shared/download-file.ts

```diff
@@ -47,6 +28,8 @@ function buildTextFileDownloader(type: string) {
   ) => downloadFile(text, type, filename, _document);
 }
 
+export const downloadJSONFile = buildTextFileDownloader('application/json');
```

Comment:
These functions boil down to binding the second argument of `downloadFile`. If that's all it does, perhaps it would be better to eliminate the indirection and just call `downloadFile(text, mimeType, filename)` directly in the code that invokes the exporters?

Reply:
Yeah, everything is clearly moving us in that direction.

I'll do it in a follow-up PR though, to simplify things here.

Reply:
Here https://github.com/hypothesis/client/pull/6098. It is based on this branch, so I'll marge it afterwards.